cli-ext/src/services/sdl/to/handler.js

```diff
@@ -42,7 +42,7 @@ const handlePayload = (payload) => {
   if (toDeriveOperation) {
     try {
       const derivation = deriveAction(toDeriveOperation, introspectionSchema, actionName);
-      const derivedActionSdl = getActionDefinitionSdl(derivation.action.name, derivation.action.arguments, derivation.action.output_type);
+      const derivedActionSdl = getActionDefinitionSdl(derivation.action.name, derivation.action.type, derivation.action.arguments, derivation.action.output_type);
```

Comment:
**[Question]** Shouldn't we also have prettier and eslint configured for `cli-ext/`?

Reply:
Should. Soon.
https://github.com/hasura/graphql-engine/issues/4364